aplijobs/birdwoot
# Feature || Add correct svg document on FileBubble

## Description

This Pr just aims to change File icon to solid


## Type of change

Please delete options that are not relevant.


## How Has This Been Tested?

- Run birdwoot
- Upload a file
- Verify new icon

<details>
<summary>New icon</summary>

![image](https://user-images.githubusercontent.com/9289461/223706767-f5ea6511-ba22-4aaf-862f-7c23e0374005.png)


</details>


<details>
<summary>Past icon</summary>

<img width="318" alt="image" src="https://user-images.githubusercontent.com/9289461/222749143-83fee004-1743-461e-80ea-a90e730af57d.png">

</details>

## Checklist:
